### ISSUE TITLE
Using AUC as main metric in classification

### ISSUE THREAD

**Issue by asdspal** [catalyst-team/catalyst#961]
#### Code
**AUC-Callback defined like this**: AUCCallback( input_key = "target")
**In runner**: main_metric = "auc/_mean"

#### Environment Details
Catalyst version: 20.09.1
PyTorch version: 1.6.0
TensorFlow version: N/A
TensorBoard version: 2.3.0
OS: Ubuntu Server 18.04
Python version: 3.7.7
CUDA runtime version: 10.2
Nvidia driver version: No CUDA
cuDNN version: 7

**Comment by Scitator** [comment]
Hi, could you please try "auc/mean"

**Comment by asdspal** [comment]
Thank you, It worked.
